File: test-export.js
const { PrismaClient } = require('@prisma/client');

const prisma = new PrismaClient();

async function testExport() {
  // Get first project
  const project = await prisma.projects.findFirst({
    select: { project_uuid: true, project_name: true }
  });

  if (!project) {
    console.error('No projects found');
    process.exit(1);
  }

  console.log('Testing export with project:', project.project_name, '(' + project.project_uuid + ')');
  
  // Make POST request to the export endpoint
  const response = await fetch('http://localhost:3000/api/export/handover-template', {
    method: 'POST',
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({
      fileName: `handover-test-${project.project_uuid}.xlsx`,
      projectUuid: project.project_uuid
    })
  });

  if (!response.ok) {
    console.error('Export failed:', response.status, response.statusText);
    const text = await response.text();
    console.error('Response:', text);
    process.exit(1);
  }

  const buffer = await response.arrayBuffer();
  console.log('Export successful, file size:', buffer.byteLength, 'bytes');
  console.log('First 20 bytes:', Buffer.from(buffer).slice(0, 20).toString('hex'));
  console.log('Content-Type header:', response.headers.get('content-type'));
  
  // Save to file for inspection
  const fs = require('fs');
  const outPath = `./export-test-${Date.now()}.xlsx`;
  fs.writeFileSync(outPath, Buffer.from(buffer));
  console.log('Saved to', outPath);

  process.exit(0);
}

testExport().catch(err => {
  console.error('Error:', err);
  process.exit(1);
});

File: test_export.py
import requests
import json

# Project UUID from the UI (Chkondideli Repair project)
project_uuid = 'a7380446-a51d-44c2-abf1-0d3a9899d3a2'

# Call the export endpoint on production
url = 'https://ice-roge3tb4l-iceerp.vercel.app/api/export/handover-template'
headers = {'Content-Type': 'application/json'}
payload = {'projectUuid': project_uuid, 'fileName': 'handover_export.xlsx'}

print(f'Calling {url}...')
try:
    response = requests.post(url, json=payload, headers=headers, timeout=30)
    print(f'Status: {response.status_code}')
    print(f'Content-Type: {response.headers.get("content-type")}')
    
    if response.status_code == 200:
        with open('handover_export.xlsx', 'wb') as f:
            f.write(response.content)
        print(f'✓ File saved: handover_export.xlsx')
        print(f'  Size: {len(response.content)} bytes')
    else:
        print(f'Error: {response.text[:200]}')
except Exception as e:
    print(f'Error: {e}')
